applications/matter_bridge/Kconfig.sysbuild

```diff
@@ -53,9 +47,16 @@ config DFU_MULTI_IMAGE_PACKAGE_APP
 config DFU_MULTI_IMAGE_PACKAGE_NET
 	default y
 
-config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
-	default y
+#### Support for WiFi patch on external flash memory.
+if BOARD_NRF5340DK_NRF5340_CPUAPP
+
+config MCUBOOT_UPDATEABLE_IMAGES
+	default 3
 
```

Comment:
isn't this missing a wifi selection?

Reply:
In this sample, we only have WiFi support, so there is no need for a wifi selection anywhere within this file.